joint_state_publisher/joint_state_publisher/joint_state_publisher.py

```diff
@@ -408,13 +457,15 @@ def main():
     # Strip off the ROS 2-specific command-line arguments
     stripped_args = rclpy.utilities.remove_ros_args(args=sys.argv)
     parser = argparse.ArgumentParser()
-    parser.add_argument('urdf_file', help='URDF file to use', nargs='?', default=None)
+    parser.add_argument('urdf_file', help='URDF file to use (deprecated, use "description_file" instead)', nargs='?', default=None)
+    parser.add_argument('description_file', help='Robot description file to use', nargs='?', default=None)
```

Comment:
I'm sorry, I made a huge mistake here.

This isn't a named option, it is just the (optional) option you pass on the command-line.  That is, it works like:

`ros2 run joint_state_publisher joint_state_publisher /path/to/urdf`

So in that case, we *can* just rename it, since the name just ends up being a local variable.  I think we should change this to:

```suggestion
    parser.add_argument('description_file', help='Robot description file to use', nargs='?', default=None)
```

Because that will just be simpler to understand.  Then we can also remove the `if not parsed_args.description_file` check below.

Reply:
That's what I was wondering when I didn't find any mention of it in the test cases. Reverting back